components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -209,11 +209,30 @@ class PageWallet extends React.Component<Props, State> {
     window.open(externalWallet.accountUrl, '_self', 'noreferrer')
   }
 
+  getAccountActivityLink = () => {
+    const { externalWallet } = this.props.rewardsData
+
+    if (!externalWallet) {
+      return undefined
+    }
+
+    if (externalWallet.status === mojom.WalletStatus.kConnected) {
+      return externalWallet.activityUrl || undefined
```

Comment:
Do we need `|| undefined` here and below?

Reply:
I guess I was playing it safe in case we have a provider now or in the future where those aren't specified.

Reply:
Yeah, those should always be non-empty. 🙂

Reply:
OK, sounds good.